common/azure_auth.go

```diff
@@ -17,9 +17,21 @@ import (
 )
 
 // List of management information
-const armDatabricksResourceID string = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
+const armDatabricksProdResourceID string = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
+const armDatabricksStagingResourceID string = "4a67d088-db5c-48f1-9ff2-0aace800ae68"
+const armDatabricksDevResourceID string = "62a912ac-b58e-4c1d-89ea-b2dbfc7358fc"
+
+func (aa *DatabricksClient) GetArmDatabricksResourceID() string {
```

Comment:
@mehdikit-db i'd suggest not to leak our internal resource UUIDs and rather supply this as configurable property through `DatabricksClient`, that gets to provider config (if mapped the right way)